plaso/multi_processing/psort.py

```diff
@@ -975,6 +979,7 @@ def AnalyzeEvents(
     if keyboard_interrupt:
       raise KeyboardInterrupt
 
+  # pylint: disable=too-many-arguments
```

Comment:
Why is this necessary? There's no change to this method, was this not being caught by the linter before for some reason? If not, please remove.

Reply:
no longer needed.